src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/snippet/form/main.js

```diff
@@ -60,7 +60,7 @@ define([
                     template: {
                         options: {
                             dropdownOptions: {
-                                url: '/admin/api/snippet/types',
+                                url: '/admin/api/snippettypes',
```

Comment:
I guess here the correct url would be `/admin/api/snippet-types`, maybe the FOSRestBundle can be configured this way?